src/index.html

```diff
@@ -11,7 +11,157 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
     <script type="text/javascript" src="./main.js"></script>
+
+    <form>
+      <fieldset class="field">
+        <legend> Personal information </legend>
+
+        <label for="Surname" class="lab">Surname:
+          <input
+            type="text"
+            id="Surname"
+            name="user"
+            required
+          >
+        </label>
+
+        <label for="Name" class="lab">Name:
+          <input
+            type="text"
+            id="Name"
+            name="userName"
+          >
+        </label>
+
+        <label for="howOld" class="lab">How old are You?
+          <input
+            type="number"
+            id="howOld"
+            name="old"
+            value="12"
+            min="1" max="100"
+          >
+        </label>
+
+        <label for="dateBirth" class="lab">Full date of birth:
+          <input
+            type="date"
+            id="dateBirth"
+            name="birth"
+            value="12"
+            min="1" max="100"
+          >
+        </label>
+
+        <label for="term" class="lab">I accept the term of the agreement
+          <input
+            type="checkbox"
+            id="term"
+            name="term"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="field-reg">
+        <legend>Registration</legend>
+
+        <label for="email" class="lab">E-mail:
+          <input
+            type="email"
+            id="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label for="Pass" class="lab">Password:
+          <input
+            type="password"
+            id="Pass"
+            name="pass"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+
+        <label for="Cats" class="lab">Do you love cats?
+          <input
+            type="radio"
+            id="Cats"
+            name="Cats"
+          > Yes
+          <input
+            type="radio"
+            id="noCat"
+            name="cat"
+          > No
+        </label>
```

Comment:
add 2 different labels for both inputs
```suggestion
        Do you love cats?
        <label class="lab">  
          <input
            type="radio"
            name="Cats"
          > 
          Yes
        </label>
        <label class="lab">
          <input
            type="radio"
            name="cat"
          >
          No
        </label>
```